docs/server/commands.rst

```diff
@@ -403,6 +403,9 @@ update_tmserver
 .. versionadded:: 2.7
 
 .. versionchanged:: 2.7.3 Renamed :option:`--overwrite` to :option:`--refresh`.
+   Also now it is possible to populate TMs using translations from specified
```

Comment:
Isn't *Now it is also possible to import translations from files.* much simpler?